cmd/az-log/README.md

```diff
@@ -0,0 +1,41 @@
+# az-log
+
+## About
+az-log is a command line tool to fetch and parse appropriate logs from the driver plugins to track the operation workflow end to end and bring better insight of driver behaviors.
+
+## Installation
+```console
+$ cd $GOPATH/src/sigs.k8s.io/azuredisk-csi-driver
+$ make install-az-log
+```
+
+## Configuration
+Check if any configuration settings need to be updated.
+```console
+$ cat $GOPATH/src/sigs.k8s.io/azuredisk-csi-driver/cmd/az-log/config/az-log.yaml
+```
+
+## Features
+
+### Source Options
+|Command|Description|
+|---|---|
+|az-log get controller |Fetch and output logs from the leader controller plugin.|
+|az-log get node \<node-name\> |Fetch and output logs from the node plugin on the given node.|
+|az-log get file \<file-name\> |Fetch and output logs from a *.log file.|
+|az-log get pod \<pod-name\>/\<container-name\> |Fetch and output logs from a specific container and plugin running for the driver.|
+
+### Retrieval Options
```

Comment:
We can mention that these options will not work with the file command.

Reply:
Yeah, thanks for the reminder.